docs/current_docs/integrations/github.mdx

```diff
@@ -0,0 +1,41 @@
+---
+slug: /integrations/github
+---
+
+# Github
+
+There are some GitHub-specific features that make the DX for working with Dagger even better.
+
+:::note
+There is a separate [integration guide for Github Actions](./github-actions.mdx)
+:::
+
+## Examples
+
+### Working with Pull Requests URLs
+
+By default the Dagger `Directory` type accepts a [remote Git repository as an input](../../cookbook#copy-a-directory-or-remote-repository-to-a-container). This makes it easy to work with the directory tree at the root of a repository or a given branch.
+
+Github contains a shorthand redirect at the `/merge` endpoint that allows you to reference the correct branch of a repository from a PR without needing to know anything about the fork or branch where the PR came from.
```

Comment:
```suggestion
GitHub contains a shorthand redirect at the `/merge` endpoint that allows you to reference the correct branch of a repository from a PR without needing to know anything about the fork or branch where the PR came from.
```

Technically, it's a ref (not a redirect) and it doesn't even point to the branch behind the PR, but the head branch merged into the base branch. That's why CIs use this ref for continuous integration: it's the _integrated_ version of the PR.

There is also a `/head` ref that points to the actual branch.